app/uk/gov/hmrc/brm/models/response/nrs/NRSStatus.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.brm.models.response.nrs
+
+import uk.gov.hmrc.brm.models.response.StatusInterface
+
+case class NRSStatus (
+                       status : Option[Int] = None,
```

Comment:
@chris-ianson Should this not be:

`case class NRSStatus(status: Int = 1, deathCode : Int = 0)` for the default?